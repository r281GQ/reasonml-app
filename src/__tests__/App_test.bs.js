// Generated by BUCKLESCRIPT VERSION 5.0.4, PLEASE EDIT WITH CARE
'use strict';

var Jest = require("@glennsl/bs-jest/src/jest.js");
var React = require("react");
var ReactTestingLibrary = require("bs-react-testing-library/src/ReactTestingLibrary.bs.js");

Jest.test("Component renders", (function (param) {
        var eta = React.createElement("div", {
              style: {
                color: "rebeccapurple"
              }
            }, React.createElement("h1", undefined, "Heading"));
        var result = ReactTestingLibrary.render(undefined, undefined, eta);
        var element = ReactTestingLibrary.getByText(/* `Str */[
              4153489,
              "Heading"
            ], undefined, result);
        return Jest.Expect[/* toMatchSnapshot */16](Jest.Expect[/* expect */0](element));
      }));

/*  Not a pure module */
