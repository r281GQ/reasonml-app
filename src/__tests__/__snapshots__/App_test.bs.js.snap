// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders 1`] = `
<h1>
  Heading
</h1>
`;
